config/software/datadog-agent-integrations.rb

```diff
@@ -50,45 +52,43 @@
       conf_directory = "../../extra_package_files/EXAMPLECONFSLOCATION"
     end
 
-    all_reqs_file = File.open("#{project_dir}/check_requirements.txt", 'w+')
-    # Manually add "core" dependencies that are not listed in the checks requirements
-    # FIX THIS these dependencies have to be grabbed from somewhere
-    all_reqs_file.puts "wheel==0.30.0 --hash=sha256:e721e53864f084f956f40f96124a74da0631ac13fbbd1ba99e8e2b5e9cafdf64"\
-        " --hash=sha256:9515fe0a94e823fd90b08d22de45d7bde57c90edce705b22f5e1ecf7e1b653c8"
-
-    all_reqs_file.close
-
-    # Install all the requirements
-    if windows?
-      pip "install -r #{project_dir}/check_requirements.txt"
-    else
-      build_env = {
-        "LD_RUN_PATH" => "#{install_dir}/embedded/lib",
-        "PATH" => "#{install_dir}/embedded/bin:#{ENV['PATH']}",
-      }
-      pip "install -r #{project_dir}/check_requirements.txt", :env => build_env
-    end
-
-    # Set frozen requirements
-    pip "freeze > #{install_dir}/agent_requirements.txt"
+  # Install all the requirements
+  if windows?
+    pip "install pip-tools==2.0.2"
```

Comment:
Can you guys use a variable up top to pin to `2.0.2`, and use string interpolation to enforce it? Just one place to define this version, please.

Reply:
pinned!